CHANGELOG.md

```diff
@@ -1531,4 +1531,41 @@ All of the merged PRs in December
 - schmidko ([@schmidko](https://github.com/schmidko))
 - rphair ([@rphair](https://github.com/rphair))
 - ubani ([@ubani](https://github.com/ubani))
-- thenic95 ([@thenic95](https://github.com/thenic95))
\ No newline at end of file
+- thenic95 ([@thenic95](https://github.com/thenic95))
+
+## 2024.04.0 (2024-01-31)
+
+All of the merged PRs in January
+
+
+#### PRs: 13
+
+- SPO node installation: Update links moved to IntersectMBO by ([@ubani](https://github.com/ubani)) in [#1187](https://github.com/cardano-foundation/developer-portal/pull/1187)
+- add liqwid to showcase by ([@thenic95](https://github.com/thenic95)) in [#1184](https://github.com/cardano-foundation/developer-portal/pull/1184)
+- Include on-chain governance section, referencing CIP-1694 on the Governance Overview Page by ([@thenic95](https://github.com/thenic95)) in [#1186](https://github.com/cardano-foundation/developer-portal/pull/1186)
+- Bump follow-redirects from 1.15.2 to 1.15.4 by ([@dependabot](https://github.com/dependabot)) in [#1188](https://github.com/cardano-foundation/developer-portal/pull/1188)
```

Comment:
`dependabot` made it into the credits again! 🤖 I am all in favour of giving credit to bots where it is due, but didn't we previously resolve to leave them out of the Changelog? 😁

Reply:
Yes we agreed on not giving credits to the bots.

Reply:
continuing in https://github.com/cardano-foundation/developer-portal/pull/1208#pullrequestreview-1865096738.